### PR TITLE
Topic/log display

### PR DESCRIPTION
**Pull request by Asakiz**

### CODE REVIEW

File: updatehub/src/main.rs [hunk 1]
```diff
@@ -138,20 +138,21 @@ async fn client_main(client_options: ClientOptions) -> updatehub::Result<()> {
...
-                        sdk::api::log::Level::Warning => "WARN",
-                        sdk::api::log::Level::Info => "INFO",
-                        sdk::api::log::Level::Debug => "DEBG",
-                        sdk::api::log::Level::Trace => "TRCE",
+                        crate::api::log::Level::Critical => "CRIT",
+                        crate::api::log::Level::Error => "ERRO",
+                        crate::api::log::Level::Warning => "WARN",
+                        crate::api::log::Level::Info => "INFO",
+                        crate::api::log::Level::Debug => "DEBG",
+                        crate::api::log::Level::Trace => "TRCE",
```
Comment (otavio) [inline updatehub/src/main.rs]: This is not needed and should not be included in the sdk change.

File: updatehub/src/mem_drain.rs [hunk 2]
```diff
@@ -199,6 +203,7 @@ mod tests {
         slog_info!(log, "{}", "info 2"; "field1" => "value1");
         slog_error!(log, "{}", "error n");
         let result = serde_json::to_string_pretty(&r_vec).unwrap();
+
```
Comment (otavio) [inline updatehub/src/mem_drain.rs:206]: Useless change.

File: updatehub/src/main.rs [hunk 3]
```diff
@@ -136,23 +136,7 @@ async fn client_main(client_options: ClientOptions) -> updatehub::Result<()> {
...
-                    };
-
-                    println!(
-                        "{timestamp} {level} {msg}",
-                        timestamp = entry.time,
-                        level = level,
-                        msg = entry.message
-                    );
-                }
+                println!("{}", response);
```
Comment (Jonathas-Conceicao) [inline updatehub/src/main.rs:139]: We shouldn't have changes to `updatehub` crate on this commit.

File: updatehub/src/mem_drain.rs [hunk 4]
```diff
@@ -35,13 +35,16 @@ impl MemDrain {
         self.logging = false;
     }
 }
-
```
Comment (Jonathas-Conceicao) [inline updatehub/src/mem_drain.rs:38]: We shouldn't be dropping this line

File: updatehub-sdk/src/api/mod.rs [hunk 5]
```diff
@@ -100,4 +97,37 @@ pub mod log {
         Debug,
         Trace,
     }
+
+    #[derive(Clone, Debug, Deserialize, Serialize)]
```
Comment (otavio) [inline updatehub-sdk/src/api/mod.rs:95]: The `Entry` struct should go just after `Log` (so before the `Level` `enum`). I did the change prior to merging it.

**Pull request merged**